packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -1109,7 +1124,7 @@ export interface SpecStep {
    * Optional snapshot version that can be additionally specified on any other
    * watch event
    */
-  watchSnapshot?: SpecSnapshotVersion;
+  watchSnapshot?: SpecWatchSnapshot;
```

Comment:
The comment ("snapshot version") is a bit out of date (maybe just change to "snapshot"?)

Reply:
Done. Updated the rest of this too.